### DISCUSSION TITLE
M68K Port?

### DISCUSSION THREAD

**Discussion by iamhumanipromise** [byteduck/duckOS#72] [Ideas]
Wondering about the feasibility of a port to M68K? So that’s MC68030, MC68040, MC68LC040 CPUs and the like, MC68020 with FPU, and so forth? 

32-bit Motorola/Freescale/NXP CPUs include: 

- MC68020 with FPU chip
- MC68EC20 with FPU chip
- MC68030 
- MC68EC030
- MC68040 
- MC68EC040
- MC68LC040
- MC68060
- MC68EC060
- MC68LC060
- Freescale/NXP Dragonball CPUs

**Comment by byteduck** [comment]
EDIT: My apologies - I assumed based on the name and a previous, similar query on the fiwix repository that this was an AI-generated post, but you do appear to be human. I'm so sorry!

Anyway, this is out of scope. duckOS is meant for the x86 architecture, and if it will be ported to any other architectures it would be ported to newer ones, probably not older ones. These old Motorola processors simply don't seem to have most of the hardware features that the kernel relies on.
